rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] A rental belongs to a book and also belongs to a person hence we would require Getters and Setters for @book and @person instance variable.  `attr_accessor :book, :person`